README.md

```diff
@@ -139,6 +139,32 @@ fab -H user@host snapshot-delete=<snapshot_id,snapshot_id1>
 ```
 
 
+### `fab snapshot-failures`
+The snapshot-failures function has 5 optional parameters: list, get, dest, show and delete. Any number of options can be given at one time, however it is best to use them one at a time to avoid any mistakes in dealing with the files.
+
+- The `snapshot-failures --list` command lists the snapshot verify failure files on the remote machine and can be downloaded through the  `--get` option.
+
+  ```
+  fab -H user@host snapshot-failures --list
+  ```
+
+- The `snapshot-failures --get [--dest]` the get command downloads the specified failure file from the remote machine locally. You can optionally specify a destination through `--dest` to download the file to and optionally rename the file, otherwise it will default to the current working directory with the original file name. 
```

Comment:
`The... the get command`

Reply:
[8e7e7a5](https://github.com/wiredtiger/testy/pull/95/commits/8e7e7a5fd171fc3a37c25cc14f5c574abb970505)